tasks/windows.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "Manage the state of Windows services (without a puppet agent)",
+  "input_method": "powershell",
+  "parameters": {
+    "action": {
+      "description": "The operation (start, stop, restart, status) to perform on the service",
+      "type": "Enum[start, stop, restart, status]"
+    },
+    "starttype": {
+      "description": "The start type (automatic, manual, disabled) to set on the service",
+      "type": "Optional[Enum[automatic, manual, disabled]]"
+    },
+    "name": {
+      "description": "The name of the service to operate on.",
```

Comment:
Related to @glennsarti comments earlier, I would specify here that this is the service `name` and call out that it is not the `displayname`, to avoid user confusion.

Reply:
Will update metadata docs